loopy/symbolic.py

```diff
@@ -256,7 +274,7 @@ def flatten(expr: ArithmeticOrExpressionT) -> ArithmeticOrExpressionT:
     return cast("ArithmeticOrExpressionT", FlattenMapper()(expr))
 
 
-class IdentityMapper(IdentityMapperBase, IdentityMapperMixin, Generic[P]):
+class IdentityMapper(IdentityMapperBase[P], IdentityMapperMixin[P], Generic[P]):
```

Comment:
Is the `Generic[P]` still needed here?

Reply:
Removed in https://github.com/inducer/loopy/compare/74665433370fb779051d2e4b2b1071aa4660a896..0ff4cf3399ed5a66f4047460f09ae09c6c6646f0